lxqt-leave/main.cpp

```diff
@@ -106,6 +106,6 @@ int main(int argc, char *argv[])
                 Qt::AlignCenter,
                 dialog.size(),
                 qApp->desktop()->screenGeometry(QCursor::pos())));
-    dialog.setMaximumSize(dialog.minimumSize());
+    dialog.setMaximumSize(dialog.size());
```

Comment:
> > ...and I think this should be changed also
> Even setting minimumSize to sizeHint() before?

Does it make any difference? (I meant to use the `dialog.sizeHint()` in the `dialog.setGeometry()` call)